visual-tests/series/comparison.js

```diff
@@ -1,88 +0,0 @@
-(function(d3, fc) {
-    'use strict';
-
-    var data = [
-        fc.data.random.financial().startDate(new Date(2014, 1, 1))(50),
-        fc.data.random.financial().startDate(new Date(2013, 12, 15))(50)
-    ];
-
-    var percentageChange = fc.indicator.algorithm.calculator.percentageChange()
```

Comment:
This is soon to be removed from d3fc-technical-indicator. Could apply the calculation here, but seems simpler to remove the test completely (given that the scale and series components are both visually-tested elsewhere)?